tests/__tests__/use-strict.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { } from 'jest';
+import { } from 'node';
+import runJest from '../__helpers__/runJest';
+
+describe('use strict', () => {
+
+  it('should show the error locations for "use strict" violations', () => {
+
+    const result = runJest('../use-strict', ['--no-cache', '-t', 'Invalid Strict']);
```

Comment:
What does `-t` flag stands for? I haven't found any doc on that, and `jest --help` haven't shown nothing also.